scripts/xcode_ios_x86_build.rb

```diff
@@ -0,0 +1,52 @@
+require 'optparse'
+require 'xcodeproj'
+
+options = {}
+option_parser = OptionParser.new do  |opts| 
```

Comment:
not familiar with ruby style, but looks like there is extra space between 'do' and '|opts|'?